src/gui/ar/groundPlaneRenderer.js

```diff
@@ -88,10 +88,10 @@ createNameSpace("realityEditor.gui.ar.groundPlaneRenderer");
 
         // create an infinite grid that fades into the distance, along the groundplane
         if (!gridHelper) {
-            const colorGrid = new THREE.Color(realityEditor.device.environment.variables.groundWireframeColor);
-            gridHelper = realityEditor.gui.threejsScene.createInfiniteGridHelper(gridSquareSizeInMm, gridRegionSizeInMm, colorGrid, maxVisibilityDistanceInMm);
+            const colorGrid = new THREE.Color('rgb(255, 240, 0)');
```

Comment:
Ideally we could still use realityEditor.device.environment.variables.groundWireframeColor, and then just set groundWireframeColor to 'rgb(255, 240, 0)' in the remote operator addon, so that if we ever want to render the groundplane in AR again it will keep its default cyan blue color, but we could go with this for now if we want to quickly merge it in for the upcoming recordings

Reply:
I tried this way, but it seems like it realityEditor.device.environment.variables.groundWireframeColor hasn't been initialized by the time we call it here. But we can fix it later.